src/providers/ObsEditProvider.js

```diff
@@ -60,18 +61,27 @@ const ObsEditProvider = ( { children }: Props ): Node => {
   ), [] );
 
   const createObservationFromGalleryPhoto = useCallback( async photo => {
-    const latitude = photo?.location?.latitude || null;
-    const longitude = photo?.location?.longitude || null;
+    const originalPhotoUri = photo?.image?.uri;
+    const firstPhotoExif = await parseExif( originalPhotoUri );
```

Comment:
When I use this branch on my Pixel 3, download the image in #259, insert

```
console.log( "firstPhotoExif: ", firstPhotoExif );
```

right after this line, and import that photo, I get
```
{"date": "2022-12-03T19:14:16.000", "latitude": 37.92129722222222, "longitude": -121.93714722222222}
```

Is that not what you're getting?

Reply:
I'm getting the exif data on Android, but I'm seeing an empty exif object for the same photo on iOS.

I'm a little confused about what's happening in the native exif parser, because it seems like we're doing a lot of date parsing in the [Android native code](https://github.com/inaturalist/react-native-exif-reader/blob/d84d462f0f36fbf1a46b7159bbcb95dd5f5d7eef/android/src/main/java/com/reactnativeexifreader/ExifReaderModule.java#L145) and [iOS native code](https://github.com/inaturalist/react-native-exif-reader/blob/d84d462f0f36fbf1a46b7159bbcb95dd5f5d7eef/ios/ExifReader.swift#L33) to make sure the date is returned in UTC/GMT format, and then we're parsing the dates again in React Native. Wouldn't it be better to either 1) return exactly what we need from native code or 2) return an unedited datetime from the native library so we can handle parsing/formatting in the client only?

Reply:
I agree that's pretty confusing and that the way the library should behave is to return the verbatim datetime in ISO8601 format, but for now, can we just take that UTC date, assume it's UTC, and convert to the local time zone?

Reply:
I can see the EXIF date for other photos in iOS (specifically, the default photos in the iOS simulator) so maybe you're right that the EXIF data is getting stripped for this photo when we add it to a simulator/device.